### ISSUE TITLE
"timestamp": "word" return empty

### ISSUE THREAD

**Issue by lvt1693** [JigsawStack/insanely-fast-whisper-api#3]
It only returns:
{
    "output": {},
    "status": "completed"
}

Do I need to disable flash attention to make it work?

**Comment by yoeven** [comment]
Do you have an example repo of the project or steps I can reproduce to get this issue? Details of the environment would be great too :)

**Comment by yoeven** [comment]
You're right, seems like there is a bug when flash attention is turned on with word time stamping, will try to fix it!
